external/avro/src/main/scala/org/apache/spark/sql/avro/package.scala

```diff
@@ -31,7 +29,6 @@ package object avro {
    *
    * @since 2.4.0
    */
-  @Experimental
   @deprecated("Please use 'org.apache.spark.sql.avro.functions.from_avro' instead.", "3.0.0")
```

Comment:
Although this is added at 2.4.0, this removal looks okay because this API is already deprecated.

cc @gengliangwang

Reply:
+1 with removing the annotation here.